ui/pages/home/home.component.js

```diff
@@ -476,6 +484,21 @@ export default class Home extends PureComponent {
             key="home-infuraBlockedNotification"
           />
         ) : null}
+        {showOutdatedBrowserWarning ? (
+          <HomeNotification
+            descriptionText={t('outdatedBrowserNotification', [
+              <span
```

Comment:
It looks like you're passing a "Here" span into the outdated browser notification message, but there aren't any substitutions in that message. This syntax is for when we're embedding something in a message.

Reply:
Thanks!